runtime/gc_include/ObjectAllocationAPI.hpp

```diff
@@ -143,7 +163,11 @@ class MM_ObjectAllocationAPI
 			{
 				/* Calculate the size of the object */
 				uintptr_t const headerSize = J9VMTHREAD_CONTIGUOUS_HEADER_SIZE(currentThread);
-				uintptr_t const dataSize = ((uintptr_t)size) * J9ARRAYCLASS_GET_STRIDE(arrayClass);
+				uintptr_t dataSize = ((uintptr_t)size) * J9ARRAYCLASS_GET_STRIDE(arrayClass);
```

Comment:
A sidenote, I don't think the `uintptr_t dataSize` argument is being used. (there is a local var of the same name)